app/components/Sidebar/components/CollectionLink.js

```diff
@@ -18,10 +18,12 @@ import CollectionSortMenu from "menus/CollectionSortMenu";
 
 type Props = {|
   collection: Collection,
-  ui: UiStore,
```

Comment:
I think you brought across some but not all of my changes here, we should either pass `ui` as a prop or take it from `useStores`

Reply:
Done. Fetching from `useStores`